### ISSUE TITLE
AdGuard for Firefox hides iframe with text area on insanelymac.com

### ISSUE THREAD

**Issue by AdamWr** [AdguardTeam/AdguardBrowserExtension#922]
### Steps to reproduce

1. Disable all fiter lists
2. Add cosmetic rule, for example: `insanelymac.com##.testtt`
3. Go here - http://www.insanelymac.com/forum/topic/330800-adguard-block-text-field-on-firefox/
4. Register/Sign in (if you want I can share you my login and password)
5. Try to write something, it is impossible, because AdGuard for Firefox hides iframe with text area

### Expected behavior

There should be text area

<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/29142494/34267576-af6e27c2-e67d-11e7-8fba-4da4c77ba715.png)
![image](https://user-images.githubusercontent.com/29142494/34267597-c21ee76c-e67d-11e7-8b2c-ffc9dc4757ae.png)

</details>

### Actual behavior

AdGuard for Firefox hides iframe with text area

<details><summary>Screenshot:</summary>

![](https://user-images.githubusercontent.com/29142494/34262050-791372c0-e66b-11e7-8ea5-0a8b8fed4e50.png)

</details>

### Your environment

* Adguard AdBlocker extension version: 2.7.3 / 2.8.1 Beta
* Environment name and version: Firefox 57.0.2
* Filters you use in Adguard: for test purpose I disabled all filters and added one rule to user filter
* Any other browser extensions you have installed: -

https://github.com/AdguardTeam/AdguardFilters/issues/8393

**Comment by telepati** [comment]
Here is the result; - with this rule, text field opened - @@||insanelymac.com^$elemhide - But, its still blocking "Choose File" button in the "More Reply Options".

After add rule;

![screen shot 2017-12-21 at 21 46 45](https://user-images.githubusercontent.com/3812286/34270399-21161a36-e699-11e7-8396-713cd6eebdcc.png)

Safari;

![screen shot 2017-12-21 at 21 52 05](https://user-images.githubusercontent.com/3812286/34270554-9f6789b0-e699-11e7-9b83-3b735379c7bd.png)

Firefox with Adguard

![screen shot 2017-12-21 at 21 44 57](https://user-images.githubusercontent.com/3812286/34270573-b166ba32-e699-11e7-9e47-4b5b98e86460.png)

**Comment by AdamWr** [comment]
Yes, I can reproduce that problem with AdGuard 2.7.3 for Firefox and it seems that it happens even if I add this website to whitelist.

Could you please check these rules (add them to the user filter):
```
@@||insanelymac.com^$generichide,important
insanelymac.com#$#html:has(>body) .cke_contents > iframe[title^="Rich text editor"][src=""] { visibility: visible!important; display: block!important; }
insanelymac.com#$#.attach_controls > iframe#iframeAttach_0 { visibility: visible!important; height: 25px!important; display: block!important; }
```
Does now all work correctly?

By the way, with AdGuard 2.8.1 Beta I can't reproduce it.

**Comment by telepati** [comment]
Ok. All buttons and text field working now. These rules are worked.

![screen shot 2017-12-22 at 16 56 57](https://user-images.githubusercontent.com/3812286/34300463-3a4fa3c8-e739-11e7-921e-c2cf429c5c55.png)

**Comment by telepati** [comment]
English filter already installed it does not the solution.

**Comment by AdamWr** [comment]
I don't know if I understand correctly what you mean, but update will be available after 1-2 hours, so please check it later.

**Comment by AdamWr** [comment]
Similar issue - https://github.com/AdguardTeam/AdguardFilters/issues/9044

**Comment by telepati** [comment]
Rules are not working anymore. I am starting get block text field on insanelymac.com again. I tried removed all rules try again but no chance. AdBlock start again blocking text field on this site.

**Comment by Mizzick** [comment]
Looks like the same issue as #880 #924, should be fixed in 2.8.2 Beta
